spec/integration/profiles_spec.rb

```diff
@@ -8,10 +8,12 @@
       fixtures :profiles
       tags 'profile'
       description 'Lists all profiles requested'
-      consumes 'application/vnd.api+json'
-      produces 'application/vnd.api+json'
       operationId 'ListProfiles'
-      parameter name: :'X-RH-IDENTITY', in: :header, schema: { type: :string }
+
+      content_types
+      auth_header
+      pagination_params
+      search_params
```

Comment:
Common things like this are now defined in `swagger_helper.rb` to DRY out the specs.